website/views/social/level-up.jade

```diff
@@ -0,0 +1,27 @@
+doctype html
+html(ng-app='habitrpg')
+  head
+    meta(charset='utf-8')
+    link(href='//fonts.googleapis.com/css?family=Lato:400,700', rel='stylesheet', type='text/css')
+    title Habitica | Your Life the Role Playing Game
+    script(type='text/javascript').
+      window.env = !{JSON.stringify(env._.pick(env, env.clientVars))};
+    != env.getManifestFiles("tmp_static_front")
```

Comment:
Adding these three lines is when `social/level-up` started redirecting to `static/front`.